c2corg_ui/static/js/map/map.js

```diff
@@ -636,11 +636,23 @@ app.MapController.prototype.handleEditModelChange_ = function(event, data) {
 
 
 /**
+ * @param {ol.Feature} feature
+ * @return {boolean}
+ * @private
+ */
+app.MapController.prototype.isLineFeature_ = function(feature) {
```

Comment:
Indeed. Since the duplicated is very small, I was not sure it was worth moving it to ``app.utils``. Do you recommend to do so?